### PR TITLE
feat(input/react): add component

### PR DESCRIPTION
**Pull request by susheelachoudry**

### CODE REVIEW

File: src/atoms/TextInput/TextInput.stories.js [hunk 1]
```diff
@@ -84,7 +86,7 @@ storiesOf('TextInput', module)
           </Space>
           <TextInput
             width={select('Width', sizeOptions, 'large', 'Common')}
-            label="Label"
+            label="Label1"
```
Comment (dhruvdutt) [inline src/atoms/TextInput/TextInput.stories.js:89]: image

Comment (dhruvdutt) [reply src/atoms/TextInput/TextInput.stories.js:89]: After entering a value, the label comes back again instead of sticking on top.

Comment (dhruvdutt) [reply src/atoms/TextInput/TextInput.stories.js:89]: image

Comment (dhruvdutt) [reply src/atoms/TextInput/TextInput.stories.js:89]: The colour of label is not correct. Also, bottom spacing of label doesn't look right.

Comment (susheelachoudry) [reply src/atoms/TextInput/TextInput.stories.js:89]: Done

Comment (dhruvdutt) [reply src/atoms/TextInput/TextInput.stories.js:89]: image

Comment (dhruvdutt) [reply src/atoms/TextInput/TextInput.stories.js:89]: Now, it's remaining active/blue even after clicking outside.

Comment (dhruvdutt) [reply src/atoms/TextInput/TextInput.stories.js:89]: Also the spacing between value and the input line border is not correct.

Comment (susheelachoudry) [reply src/atoms/TextInput/TextInput.stories.js:89]: >Now, it's remaining active/blue even after clicking outside.
Isnt this required ?

>Also the spacing between value and the input line border is not correct. Line height.
I am unable to get exact spacing from figma.

Comment (dhruvdutt) [reply src/atoms/TextInput/TextInput.stories.js:89]: No, look at Figma. Blue is there only in active state.

**Comment by susheelachoudry** [comment]
closing this PR since added input component in a shield repo.
https://github.com/razorpay/frontend-commander/pull/98

**Pull request closed without merging**